### ISSUE TITLE
Knight problem - imprecise task description

### ISSUE THREAD

**Issue by cytrowski** [Jarosh/Exercises#3]
My issue is related to that comments thread: https://github.com/Jarosh/Exercises/issues/2#issuecomment-273329599

Task description does not contain a restriction on knight movements as far as I see. It means there is no phrase telling me that I cannot go [-2, 1] after goint [1, 2]. That way I can reach any field on the infinite board.

Can you explain me why did you made different assumption in your solution? @Jarosh Maybe I don't see something important there?

**Comment by seyfer** [comment]
+1, need to be explained

@cytrowski have you solved a problem with a backward moves also? Could you share your solution?

**Comment by cytrowski** [comment]
@seyfer Yes I did it some time ago however I remember it being rejected - I've never figured out how come there are fields the knight cannot reach in finite number of moves.

**Comment by seyfer** [comment]
@cytrowski try BFS algorithm ;)
